js/app.js

```diff
@@ -0,0 +1,21 @@
+$(function(){
+  $('#stopButton').click(illuminateRed);
+  $('#slowButton').click(illuminateYellow);
+  $('#goButton').click(illuminateGreen);
+
+  function illuminateRed() { 
+    $(".bulb").css("backgroundColor" , "black");
```

Comment:
@jiniekwak this could have been made a function that could have called by illuminate functions